src/repl-frontend/scala/tools/nsc/interpreter/jline/Reader.scala

```diff
@@ -54,8 +54,7 @@ object Reader {
 
     System.setProperty(LineReader.PROP_SUPPORT_PARSEDLINE, java.lang.Boolean.TRUE.toString())
 
-    //val terminal = TerminalBuilder.builder().build()
-    val terminal  = TerminalBuilder.terminal() // defaults for now
+    val terminal = TerminalBuilder.builder().jna(true).build()
```

Comment:
If JNA is optional, then this change I made might no longer be good?
https://github.com/jline/jline3/blob/master/terminal/src/main/java/org/jline/terminal/TerminalBuilder.java

Reply:
It's only intended to be completely optional for people who are using scala-compiler.jar without having any interest in the REPL at all. So I think it's okay.